drivers/i2s/Kconfig.stm32

```diff
@@ -0,0 +1,67 @@
+# Kconfig - STM32 I2S driver configuration options
+#
+# Copyright (c) 2018 STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig I2S_STM32
+	bool "STM32 MCU I2S controller driver"
+	depends on I2S && SOC_SERIES_STM32F4X
+	select DMA
+	default n
+	help
+	  Enable I2S support on the STM32 family of processors.
```

Comment:
Please indicate here on which series it was tested

Reply:
ack